### PR TITLE
EMPs will generally fuck up clockwork structures

### PR DESCRIPTION
**Pull request by ChangelingRain**
:cl: Joan
rscadd: EMPs will generally fuck up clockwork structures.
/:cl:

Geis bindings will break, powered structures will disable and consume power, clockwork obelisks will cut off gateways, interdiction lenses will disable for a long time, caches and ocular wardens will force-unanchor and take some damage, and anima fragments will slow down briefly. Cogscarabs no longer actually DIE to emps, but are still stunned by them.

Basically, I want you to throw EMP grenades at the machine cult, because that seems like a thing you should be able to do, right?

### CODE REVIEW

**Comment by RandomMarine** [comment]
You'd think a clockwork cult, using machines that work with gears and mechanisms and shit wouldn't be affected by EMPs, which mostly just fucks up electronics.

**Comment by coiax** [comment]
We have fictional EMPs in tgstation anyway, they can do that sort of shit because that's the rules of fiction

**Comment by francinum** [comment]
you do realize how easy EMPs are to aquire, right?

**Comment by AdamElTablawy** [comment]
you do realize that weapons are easier to acquire, right?

nobody uses EMP's generally unless it's against borgs and most of the time raiding a clock cult base is usually just walls of laser spam anyway so it's not like this is going to be a significant balance change

it's not like making EMPs something that affects a machine cult something that's going to make them underpowered.

and EMP's are fairly hard to acquire anyway. There's the metastation EMP box and sometimes the armory, and the ion rifle, and that's about it unless someone does the uranium-iron thing and chemistry never gets uranium anyway.

**Comment by AdamElTablawy** [comment]
tbh it doesn't seem much like EMPs will help against clock cultists at the moment even with this nerf. It'd be better if it damaged all types of clock cult machines and it'd actually have an impact if it damaged the global component cache itself and made it lose some components.

**Comment by SgtxRootbeer** [comment]
This seems to also buff bloodcult on the rare occasion that both cults are present.

**Comment by AdamElTablawy** [comment]
Because bloodcult has an emp so an emp buff is a bloodcult buff right lol xd

That's like reducing armor values on clockcult armor snd then saying it's a bloodcult buff because bloodcult can do damage

Come the hecking fuck on sgtrootbeer

**Comment by SgtxRootbeer** [comment]
The autism... It's almost tangible.
Bloodcult is the only source of a EMP that covers the entirety of the station (some larger stations excluded.) And the source of a lot of easy-to-get emps (talisman). So, although it is possible to acquire a EMP outside of bloodcult, bloodcult takes usage of an EMP to extreme levels.

This is more like increasing the damage bloodcultists do to clockcultists by a factor of 10. 
(Although we do have similar code which isn't quite as extreme and works both ways.)

**Comment by jughu** [comment]
This is a MINERS GET URANIUM episode if it gets merged
